pkg/imager/imager.go

```diff
@@ -349,6 +349,12 @@ func (i *Imager) buildCmdline() error {
 		cmdline.Append("console", "tty0")
 	}
 
+	if quirks.New(i.prof.Version).SupportsSELinux() {
+		// Talos 1.9 introduces SELinux in permissive mode
+		cmdline.Append("selinux", "1")
+		cmdline.Append("enforcing", "0")
```

Comment:
How do we handle the next update where we're going to set `enforcing=1`? Will we be able to ask installer to override `enforcing`? Or maybe we should omit enforcing option for now at all, as kernel default is permissive

Reply:
if we can omit, let's omit for now.